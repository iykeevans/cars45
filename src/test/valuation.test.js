import { render } from "./test-utils";

import Valuation from "../pages/valuation";

describe("Valuation page", () => {
  test("Valuation page matches snapshot", () => {
    const { asFragment } = render(<Valuation />);
    expect(asFragment()).toMatchSnapshot();
  });
});
