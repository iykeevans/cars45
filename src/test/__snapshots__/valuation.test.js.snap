// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Valuation page Valuation page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="header-one"
  >
    <div
      class="container"
    >
      <div
        class="row justify-content-md-end"
      >
        <div
          class="col-10 col-lg-4 phone-box"
        >
          <div
            class="top-phone"
          >
            <img
              alt="phone"
              src="/assets/icons/phone.svg"
            />
            <p
              class="darkgrey-color"
            >
              +234 818 984 0160
            </p>
            <div
              class="divider"
            />
            <p
              class="darkgrey-color"
            >
              +234 818 984 0160
            </p>
          </div>
        </div>
        <div
          class="col-1 col-md-1 profile"
        >
          <div
            class="btn-group"
          >
            <button
              aria-expanded="false"
              aria-haspopup="true"
              class="btn btn-outline-secondary dropdown-toggle"
              data-toggle="dropdown"
              type="button"
            >
              <img
                alt="profile"
                src="/assets/icons/profile-icon.svg"
              />
            </button>
            <div
              class="dropdown-menu"
            >
              <button
                class="dropdown-item btn btn-link darkgrey-color"
              >
                Sign In
              </button>
              <div
                class="dropdown-divider"
              />
              <button
                class="dropdown-item btn btn-link darkgrey-color"
              >
                Register
              </button>
            </div>
          </div>
        </div>
      </div>
      <nav
        class="navbar navbar-expand-lg navbar-light bg-light"
      >
        <a
          class="navbar-brand"
          href="/"
        >
          <img
            alt="logo"
            class="logo"
            src="/assets/icons/logo.svg"
          />
        </a>
        <button
          aria-controls="navbarNavDropdown"
          aria-expanded="false"
          aria-label="Toggle navigation"
          class="navbar-toggler"
          data-target="#navbarNavDropdown"
          data-toggle="collapse"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
        <div
          class="collapse navbar-collapse"
          id="navbarNavDropdown"
        >
          <ul
            class="navbar-nav"
          >
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/brandnew"
              >
                Brand New Cars 
              </a>
            </li>
            <li
              class="nav-item dropdown"
            >
              <button
                aria-expanded="false"
                aria-haspopup="true"
                class="nav-link dropdown-toggle btn btn-outline-secondary used"
                data-toggle="dropdown"
                id="navbarDropdownMenuLink"
                role="button"
              >
                Used Cars
              </button>
              <div
                aria-labelledby="navbarDropdownMenuLink"
                class="dropdown-menu used-menu"
              >
                <a
                  class="dropdown-item"
                  href="/buy"
                >
                  Buy
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/sell"
                >
                  Sell
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/swap"
                >
                  Swap
                </a>
              </div>
            </li>
            <li
              class="nav-item dropdown"
            >
              <button
                aria-expanded="false"
                aria-haspopup="true"
                class="nav-link dropdown-toggle btn btn-outline-secondary inspection"
                data-toggle="dropdown"
                id="navbarDropdownMenuLink"
                role="button"
              >
                Inspection Services
              </button>
              <div
                aria-labelledby="navbarDropdownMenuLink"
                class="dropdown-menu inspect"
              >
                <a
                  class="dropdown-item"
                  href="/ride-hailing"
                >
                  Ride Hailing 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/private-individual"
                >
                  Private/Individual 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/premium-inspection"
                >
                  Premium Inspection 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/valuation"
                >
                  Valuation 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/due-dilligence"
                >
                  Due Dilligence 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/all-in-one"
                >
                  All-in-one 
                </a>
              </div>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/loan"
              >
                Car Loan 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/service"
              >
                Service & Repair 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/franchise"
              >
                Become a Franchise 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/corporate"
              >
                Corporate Service 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/import"
              >
                Import Cars 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/referral"
              >
                Referral Program 
              </a>
            </li>
            <li
              class="nav-item dropdown"
            >
              <button
                aria-expanded="false"
                aria-haspopup="true"
                class="nav-link dropdown-toggle btn btn-outline-secondary"
                data-toggle="dropdown"
                id="navbarDropdownMenuLink"
                role="button"
              >
                Other Services
              </button>
              <div
                aria-labelledby="navbarDropdownMenuLink"
                class="dropdown-menu"
              >
                <a
                  class="dropdown-item"
                  href="/about"
                >
                  About Us 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/check"
                >
                  Background Check 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/dealer"
                >
                  Become A Dealer 
                </a>
              </div>
            </li>
          </ul>
        </div>
      </nav>
    </div>
  </div>
  <div
    class="ride-hailing"
  >
    <div
      class="jumbotron"
    >
      <div
        class="feedback-button"
      >
        <a
          class="d-none"
          href="https://docs.google.com/forms/d/1uvqqKDzYS7pOVG1wKOoyNpVeyYTJpE1mXXYh0-y-IdE/edit"
          id="survey-go"
          target="_blank"
        >
          Survey link
        </a>
        <button
          class="btn btn-danger"
        >
          Feedback
        </button>
      </div>
    </div>
    <div
      class="get-inspected"
    >
      <div
        class="container"
      >
        <div />
        <div
          class="row"
        >
          <div
            class="col-md-4 inspect-img d-none d-lg-block"
          >
            <img
              alt="inspect"
              src="/assets/images/inspection.png"
            />
            <p>
              Determine the current market value of the vehicle
            </p>
          </div>
          <div
            class="col-lg-8"
          >
            <h2
              class="inspect-header text-center"
            >
              Get your car inspected
            </h2>
            <div
              class="row ml-4 mb-4"
            >
              <div
                class="col-md-2 inspect-info-img"
              >
                <img
                  alt="book"
                  class="book"
                  src="/assets/icons/book.svg"
                />
              </div>
              <div
                class="col-md-6 details align-self-center"
              >
                <p>
                  Book an inspection right here or walk into any of our centers
                </p>
              </div>
            </div>
            <div
              class="row ml-4 mb-4"
            >
              <div
                class="col-md-2 inspect-info-img"
              >
                <img
                  alt="book"
                  class="book"
                  src="/assets/icons/money.svg"
                />
              </div>
              <div
                class="col-md-6 details align-self-center"
              >
                <p>
                  Make a deposit of ₦10,000 at your inspection center of choice.
                </p>
              </div>
            </div>
            <div
              class="row ml-4 mb-4"
            >
              <div
                class="col-md-2 inspect-info-img"
              >
                <img
                  alt="book"
                  class="book"
                  src="/assets/icons/inspect-done.svg"
                />
              </div>
              <div
                class="col-md-6 details align-self-center"
              >
                <p>
                  The inspection is carried within an hour.
                </p>
              </div>
            </div>
            <div
              class="row ml-4 mb-4"
            >
              <div
                class="col-md-2 inspect-info-img"
              >
                <img
                  alt="book"
                  class="book"
                  src="/assets/icons/inspection-report.svg"
                />
              </div>
              <div
                class="col-md-6 details align-self-center"
              >
                <p>
                  Receive your inspection result within 30 minutes later!
                </p>
              </div>
            </div>
            <button
              class="btn btn-secondary orange-button mt-5 mb-5"
            >
              Book An Inspection
            </button>
          </div>
        </div>
      </div>
      <div
        class="col-md-12"
      >
        <div
          class="bottom-div"
        >
          <img
            alt="security"
            src="/assets/icons/security.svg"
          />
          <p>
            Cars45 is commited to keeping your information safe
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    class="footer-one"
  >
    <div
      class="footer-section-one"
    >
      <div
        class="container"
      >
        <div
          class="row"
        >
          <div
            class="col-12 col-md-4 col-lg-3"
          >
            <img
              alt="logo"
              class="logo"
              src="/assets/icons/logo-white.svg"
            />
          </div>
          <div
            class="col-6 col-md-4 col-lg-3"
          >
            <ul>
              <li>
                <a
                  href="/"
                >
                  About Us
                </a>
              </li>
              <li>
                <a
                  href="/"
                >
                  Contact Us
                </a>
              </li>
              <li>
                <a
                  href="/"
                >
                  Our Services
                </a>
              </li>
              <li>
                <a
                  href="/"
                >
                  Careers
                </a>
              </li>
              <li>
                <a
                  href="/"
                >
                  Blog
                </a>
              </li>
            </ul>
          </div>
          <div
            class="col-6 col-md-4 col-lg-3"
          >
            <ul>
              <li>
                <a
                  href="/"
                >
                  Inspection Centers
                </a>
              </li>
              <li>
                <a
                  href="/"
                >
                  Auction Locations
                </a>
              </li>
              <li>
                <a
                  href="/"
                >
                  Required Documents
                </a>
              </li>
              <li>
                <a
                  href="/"
                >
                  Common Questions
                </a>
              </li>
              <li>
                <a
                  href="/"
                >
                  Background Check
                </a>
              </li>
            </ul>
          </div>
          <div
            class="col-12 col-lg-3 align-self-center"
          >
            <div
              class="socials"
            >
              <a
                href="http://facebook.com/cars45.NG"
                target="_blank"
              >
                <img
                  alt="facebook"
                  class="facebook"
                  src="/assets/icons/facebook.svg"
                />
              </a>
              <a
                href="https://www.instagram.com/cars45ng/"
                target="_blank"
              >
                <img
                  alt="instagram"
                  src="/assets/icons/instagram.svg"
                />
              </a>
              <a
                href="https://twitter.com/cars45ng"
                target="_blank"
              >
                <img
                  alt="twitter"
                  src="/assets/icons/twitter.svg"
                />
              </a>
              <a
                href="/youtube.com"
                target="_blank"
              >
                 
                <img
                  alt="youtube"
                  src="/assets/icons/youtube.svg"
                />
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="footer-section-two"
    >
      <div
        class="container"
      >
        <div
          class="row"
        >
          <div
            class="col-md-6"
          >
            <ul>
              <li>
                <a
                  href="/"
                >
                  Terms & Conditions
                </a>
              </li>
              <li>
                <a
                  href="/"
                >
                  Privacy
                </a>
              </li>
            </ul>
          </div>
          <div
            class="col-md-6 text-center text-md-right"
          >
            <p>
              2021Cars45.com. ©All rights reserved
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
